### ISSUE TITLE
 jQuery minimum version requirement since 0.8.2

### ISSUE THREAD

**Issue by ngavard** [flot/flot#1240]
Hi,
We are using Richfaces 3.3.3 that ship jquery 1.3.2.
In the flot's readme, it is said :
You need at least jQuery 1.2.6, but try at least 1.3.2 for interactive charts because of performance improvements in event handling.
That seems good to me :)
But, I'm just upgrading flot from 0.7 to 0.8.2 and it seems that there is a call to element.detach(). Can you confirm this is a function from jQuery 1.4 ?

If so, I need to know if there is a chance to see the backward compatibility be restored to 1.3.2 (at least) or if I have to upgrade my components. 

Thanks for any comment.
